Git Formats/tests/syntax_test_git_log

```diff
@@ -74,3 +74,45 @@ Date:   Thu Sep 21 22:53:04 2017 +0200
 #^^^ keyword.other.header.git.log
 #   ^ punctuation.separator.mapping.pair.git.log - keyword.other.header.git.log
 #       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ string.unquoted.log
+
+commit 344d003a330e244cc507d17c2f224bb295013511 (HEAD -> git_show, sublimehq/master, master)
+Author: Thomas Smith <thom@thomthesmith.com>
+Date:   Tue Aug 24 13:44:17 2021 -0400
+
+    [TypeScript] [TSX] Fix arrow function type parameter tests for TSX. (#2987)
+
+diff --git a/JavaScript/TypeScript.sublime-syntax b/JavaScript/TypeScript.sublime-syntax
+#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.shell
+#                                                                                       ^ - source.shell
+#^^^ source.shell meta.function-call.identifier variable.function
+#    ^^ punctuation.definition.parameter
+#      ^^^ variable.parameter.option
+index ae1b1061..c1a585d0 100644
```

Comment:
`index ...` is not covered by `source.diff`. Would we want to scope it in this syntax?

Reply:
yep, makes sense to :) done

Reply:
Would it make sense to add a meta scope such as `meta.diff.header.extended.git.log` to the whole extended header section?